CRUD for items in Ember

ariejan opened the pull request:
This is very much a work-in-progress, there are no Ember tests yet and all help is greatly appreciated.
- [x] List of items
- [x] Clean subdomain separation of responsibilities and CORS
- [x] Handle URL items

ariejan committed:
Update dashboard URL in readme

ariejan committed:
Nice item overview

ariejan commented:
@rachid @RoelN @eirvandelden @bazzel feel free to check out the feature branch, your code, input and help is greatly appreciated!

eirvandelden commented:
@ariejan Do you want to add pagination by default instead of waiting until we actually need it?

ariejan commented:
The API does not support it yet, but I don't think downloading the entire set of items is a good idea. Maybe endless scrolling, or some other solution is also possible. We might just leave it for this PR.

ariejan committed:
Merge branch 'master' into ember-items-crud

ariejan committed:
Update test for changed layout

ariejan committed:
Use subdomains/CORS correctly.

TLD for serving content.
admin.TLD for serving Ember.js app
api.TLD for serving the API

Fire Dragon provides built-in CORS to make sure
the Ember.js app can connect to api.TLD.

ariejan committed:
Save go dependencies

ariejan committed:
Implement URL redirect

ariejan committed:
Handle not found items
